src-docs/src/views/description_list/description_list_example.js

```diff
@@ -239,13 +239,26 @@ export const DescriptionListExample = {
         },
       ],
       text: (
-        <p>
-          Using the <EuiCode>align</EuiCode> and <EuiCode>compressed</EuiCode>{' '}
-          props you can further tailor the look of a description list. This
-          works with column and inline types. You can also adjust the{' '}
-          <EuiCode>gutterSize</EuiCode> prop to increase and decrease vertical
-          spacing between <EuiCode>EuiDescriptionList</EuiCode> elements.
-        </p>
+        <>
+          <h4>Alignment & Compression</h4>
+          <p>
+            Utilize the <EuiCode>align</EuiCode> and
+            <EuiCode>compressed</EuiCode> props to refine the presentation of
+            your description list. These are compatible with both column and
+            inline types.
+          </p>
+          <h4>Vertical spacing</h4>
+          <p>
+            Modify the <EuiCode>gutterSize</EuiCode> prop to control the
+            vertical spacing between <EuiCode>EuiDescriptionList</EuiCode>{' '}
+            elements.
```

Comment:
Optional suggestion:
```suggestion
            elements. This will not affect inline list types.
```